build/dsc/manager.rb

```diff
@@ -32,9 +32,6 @@ def initialize
         'MSFT_Credential' => { 'user' => 'user', 'password' => 'password' },
         'MSFT_KeyValuePair' => { 'somekey' => 'somevalue' },
         'MSFT_KeyValuePair[]' => { 'somekey' => 'somevalue', 'somekey2' => 'somevalue2' },
-        # these hashes will be automatically wrapped in tests
-        'MSFT_xFileDirectory[]' => { 'DestinationPath' => 'c:/foo/bar', 'Recurse' => true },
-        'MSFT_xWebBindingInformation[]' => { 'Port' => 8080, 'Protocol' => 'https' },
```

Comment:
Last commits generates EmbeddedInstance spec values on the fly... no more need to statically define these.